app/models/absence_request.rb

```diff
@@ -3,7 +3,23 @@
 #  - attributes are defined on Request
 #  - behavior can be defined here
 class AbsenceRequest < Request
+  include AasmConfig
+
   def to_s
     "#{creator} Absence"
   end
+
+  aasm column: "status" do
+    # add in an additional states pending_cancelation & recorded which are only valid for an absence request
+    state :pending_cancelation
+    state :recorded
```

Comment:
The diagram labels this status as "reported". Did we change it to recorded and not update that diagram?

Reply:
I'll update the diagram.  There was a ticket to change it https://github.com/pulibrary/approvals/issues/227